bundle-workflow/Jenkinsfile

```diff
@@ -112,6 +105,11 @@ pipeline {
 
 void build() {
     git url: 'https://github.com/opensearch-project/opensearch-build.git', branch: 'main'
+
+    // psutil does not support arm64 yet, and it is not needed for build and assamble on Jenkins
+    // https://github.com/giampaolo/psutil/issues/1966
+    sh "sed -i '/psutil/d' bundle-workflow/Pipfile"
```

Comment:
We should revert the requirement for the library if it does not work in our supported platforms @gaiksaya

Reply:
💯 Let's open an issue at least.

Reply:
Sure @peternied @dblock also psutil does not work on Windows with cygwin.

Reply:
https://github.com/opensearch-project/opensearch-build/issues/597

Reply:
It works on arm64. See issue https://github.com/opensearch-project/opensearch-build/issues/527
Not sure about windows. If anyone has any work around I am all ears

Reply:
Correct me if wrong @gaiksaya if we want parent pid here: https://github.com/opensearch-project/opensearch-build/blob/main/bundle-workflow/src/test_workflow/integ_test/local_test_cluster.py#L116. Can't we use `os.getppid()`?

Reply:
More than that we want children process id here: https://github.com/opensearch-project/opensearch-build/blob/main/bundle-workflow/src/test_workflow/integ_test/local_test_cluster.py#L118

Reply:
For that something like: 
```shell
ps_output = run(['ps', '-opid', '--no-headers', '--ppid', str(os.getpid())],
                stdout=PIPE, encoding='utf8')
child_process_ids = [int(line) for line in ps_output.stdout.splitlines()]
``

Reply:
We're going to have a lot of other things broken on Windows, let's take that on separately?

Reply:
Already added python3-devel and close the issue.